third_party/spirv-cross/spirv_glsl.cpp

```diff
@@ -14977,7 +14977,11 @@ string CompilerGLSL::flags_to_qualifiers_glsl(const SPIRType &type, const Bitset
 	{
 		auto &execution = get_entry_point();
 
-		if (flags.get(DecorationRelaxedPrecision))
+		if (type.basetype == SPIRType::UInt && is_legacy()) {
```

Comment:
Are these changes submitted to spirv-cross upstream? Otherwise, we'll lose these patches when building inside of G3

Reply:
Submitted an upstream PR, but I'm not sure they'll like it since it doesn't fix the root of the problem.